### PR TITLE
Feature/setup refactoring

### PR DESCRIPTION
**Pull request by dev-tim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,11 +2,16 @@
   "name": "controlcenter",
   "version": "0.0.0",
   "dependencies": {
-    "react": "^0.12.2"
+    "es5-shim": "^4.1.0",
```
Comment (mr-mig) [inline package.json]: If it is required by reflux, why is it added as a high-level dependency to the project itself?

File: public/app/common/regions/navigation/nav-item/nav-item.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+var React = require("react");
+var styles = require("./style.scss");
```
Comment (mr-mig) [inline public/app/common/regions/navigation/nav-item/nav-item.jsx]: I think you have messed the styles already. This line in html is not needed:

**Comment by mr-mig** [comment]
I would say this PR is very large. 
Can you please make smaller pull requests in the future?

File: package.json (hunk 1 above)
Comment (dev-tim) [inline package.json]: "Like React, Reflux depends on an es5-shim for older browsers."

And yep it's not included as a dependency in reflux

File: public/app/common/regions/navigation/nav-item/nav-item.jsx (hunk 2 above)
Comment (dev-tim) [inline public/app/common/regions/navigation/nav-item/nav-item.jsx]: It's not needed in case if you go with "styles attached to head"  way.

**Comment by dev-tim** [comment]
Yep, I agree it's VERY large, however there is half of the project changed.

**Pull request merged**